### PR TITLE
bump kube to 1.23.1

### PR DESCRIPTION
**Pull request by s-urbaniak**
/cc @stlaz @p0lyn0mial

### CODE REVIEW

**Comment by s-urbaniak** [comment]
holding for final review

File: cmd/oauth-apiserver/main.go [hunk 1]
```diff
@@ -33,10 +23,8 @@ func main() {
 	}
 
 	command := NewOAuthAPIServerCommand()
-	if err := command.Execute(); err != nil {
-		fmt.Fprintf(os.Stderr, "%v\n", err)
-		os.Exit(1)
-	}
+	code := cli.Run(command)
+	os.Exit(code)
```
Comment (p0lyn0mial) [inline cmd/oauth-apiserver/main.go:27]: nit: we could change it to `os.Exit(cli.Run(NewOAuthAPIServerCommand()))`

**Pull request merged**